### PR TITLE
Prelanding/fix binary data

### PR DESCRIPTION
**Pull request by AlvaroVega**
from https://github.com/telefonicaid/iotagent-json/pull/736

### CODE REVIEW

**Review by mapedraza — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
Discussing this with @mapedraza we think it would be great to include documentation in this PR (at the present moment, it seems this is not document at all).

**Comment by mapedraza** [comment]
> Discussing this with @mapedraza we think it would be great to include documentation in this PR (at the present moment, it seems this is not document at all).

Addressed in https://github.com/telefonicaid/iotagent-json/pull/760/commits/a03ac1a16e02bbbb57b9b97be2e0b44ebcae7bfc

File: docs/usermanual.md [hunk 1]
```diff
@@ -87,7 +87,21 @@ It is possible to send a single measure to IoT Platform using an HTTP POST reque
...
+
+```json
+{
+    ...
+    "attrHex":"6869"
+    ...
+}
+```
+
+Which `68` correspond to the character `h` and `69` corresponds to `i`  ASCII enconded.
```
Comment (fgalan) [inline docs/usermanual.md:104]:
```suggestion
For a measure sent to `/iot/json/attrs/attrHex` with value `hello` the resulting attribute value persisten in the context
broker will be:

```json
{
    ...
    "attrHex":"68656c6c6f"
    ...
}
```

where `68` corresponds to the character `h`, `65` corresponds to `e`, `6c` corresponds to `l` and `6f` corresponds to `o` in ASCII code.
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**